etl/steps/data/garden/faostat/2024-03-14/additional_variables.meta.yml

```diff
@@ -670,3 +670,14 @@ tables:
   # All metadata for maize_and_wheat and fertilizer_exports is prepared via script.
   # maize_and_wheat:
   # fertilizer_exports:
+  net_exports_as_share_of_supply:
+    variables:
+      net_exports_as_share_of_supply:
+        title: Net food trade as a share of domestic supply
+        unit: "%"
+        short_unit: "%"
+        description_short: |
+          Net exports, calculated as the difference between exports and imports, expressed as a share of domestic supply. This includes all agricultural products, such as food allocated for human consumption, animal feed, and other uses.
```

Comment:
Clarifying that the exports and imports are agricultural products earlier in the description would be useful.